go.mod

```diff
@@ -270,3 +270,5 @@ replace (
 	github.com/pingcap/tidb/parser => ./parser
 	go.opencensus.io => go.opencensus.io v0.23.1-0.20220331163232-052120675fac
 )
+
+replace github.com/tikv/client-go/v2 => github.com/crazycs520/client-go/v2 v2.0.0-alpha.0.20240516063820-56026663209d
```

Comment:
Should this be changed to the client-go with tidb-6.5-with-kv-timeout branch?

Reply:
Oh it seems the client-go PR is not merged yet.

Reply:
The client-go PR is merged.

Reply:
done.